### PR TITLE
Add Travis-CI continous integration support

### PR DESCRIPTION
**Pull request by chatelao**
Allow the software to be built automatically over at "travis-ci.org".

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+language: java
+
+#
+# INSTALL: setup the software, i.e. build & install
+#
+install:
+  - mvn clean install -DskipTests --fail-at-end
```
Comment (IlyaAvdeev) [inline .travis.yml:19]: sometimes `-DskipTests` doesn't work properly (only surefire 2.3)
we use older version of it, so I guess this approach won't work. Please fix

**Review by IlyaAvdeev — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (IlyaAvdeev) [reply .travis.yml:19]: @chatelao will you do any changes/respond the comment above?

**Comment by modax** [comment]
I'm afraid travis config won't be that simple. See https://github.com/SmartBear/soapui/pull/449 which I have just created.

File: .travis.yml (hunk 1 above)
Comment (chatelao) [reply .travis.yml:19]: which versions do you use?

Comment (IlyaAvdeev) [reply .travis.yml:19]: `which versions do you use?`
2.12.1
@chatelao have you seen #449 PR? it looks like you and @modax are working on the same things.

File: soapui/pom.xml [hunk 2]
```diff
@@ -203,13 +203,13 @@
...
             <artifactId>analytics-core</artifactId>
-            <version>5.5.0-SNAPSHOT</version>
+            <version>5.5.0-AT</version>
         </dependency>
 
         <dependency>
             <groupId>com.smartbear.utils.analytics</groupId>
             <artifactId>out-app-analytics-provider</artifactId>
-            <version>5.5.0-SNAPSHOT</version>
+            <version>5.5.0-AT</version>
```
Comment (IlyaAvdeev) [inline soapui/pom.xml:212]: it looks like the PR is broken, current "next" branch is based on 5.5.0 version. These 2 should not be changed.

**Comment by IlyaAvdeev** [comment]
Didn't get any response. Closing the PR

**Pull request closed without merging**